src/multiselect_controls.js

```diff
@@ -427,6 +430,17 @@ export class MultiselectControls {
       wrappedFunc.isWrapped = true;
       return wrappedFunc;
     })(Blockly.Gesture.prototype.handleBlockStart);
+    Blockly.Gesture.prototype.setStartBlock = (function(func) {
+      if (func.isWrapped) {
+        return func;
+      }
+
+      const wrappedFunc = function(block) {
+
+      };
+      wrappedFunc.isWrapped = true;
+      return wrappedFunc;
+    })(Blockly.Gesture.prototype.setStartBlock);
```

Comment:
What's the purpose here? You intend to monkey patch the `Blockly.Gesture.prototype.setStartBlock` but have an empty `wrappedFunc`, is there anything missing here?

Reply:
> I also deactivated the handleBlockstart and setStartBlock gestures, so those should not be causing a problem either

Ah okay, there's a clear way to write this if you want to set this as empty.

Reply:
Something like this should be enough I guess

```suggestion
Blockly.Gesture.prototype.setStartBlock = function(block) {}
```

Reply:
I removed the setStartBlock one and left the handleBlockStart (as setStartBlock is called in handleBlockStart). This is only called when multiselect mode is turned on. I think this is kind of related to the thing that Beka talked about (pointer down event listener) and helps avoid a bug when selecting blocks in multiselect mode by clicking.

Reply:
> I removed the setStartBlock one and left the handleBlockStart (as setStartBlock is called in handleBlockStart). This is only called when multiselect mode is turned on. I think this is kind of related to the thing that Beka talked about (pointer down event listener) and helps avoid a bug when selecting blocks in multiselect mode by clicking.

I'm not sure this will cause problems, we still need to be careful about setting something to empty by monkey patch, I'll check.

Reply:
Related PR if we remove this: https://github.com/google/blockly/pull/1590

Reply:
Looks like I can't find any issue when setStartBlock is removed, but it should have some use here or there (or why don't we just send a PR there and remove it?), and it would be good to double-check this with the Blockly team (that's another reason I don't like monkey patch)